src/appMain/hmi_capabilities.json

```diff
@@ -504,202 +504,202 @@
                               {
                                  "densityAvailable":true,
                                  "name":"FRONT_LEFT_HIGH_BEAM",
-                                 "sRGBColorSpaceAvailable":true
+                                 "RGBColorSpaceAvailable":true
```

Comment:
this shall be `rgbColorAvailable`

Reply:
Unless you want to make a new proposal to alter the name, the intent of the name was `rgbColorSpaceAvailable`